hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestAlterTableDropPartition.scala

```diff
@@ -81,13 +81,13 @@ class TestAlterTableDropPartition extends HoodieSparkSqlTestBase {
   }
 
   Seq(false, true).foreach { urlencode =>
-    test(s"Drop single-partition table' partitions, urlencode: $urlencode") {
+    test(s"Drop single-partition table's partitions, urlencode: $urlencode") {
       withTempDir { tmp =>
         val tableName = generateTableName
         val tablePath = s"${tmp.getCanonicalPath}/$tableName"
 
         import spark.implicits._
```

Comment:
It makes no sense to config urlencode=false when partition value contains slash -- '/' , it brings much obstacles when parsing partition values from path, say when executing 'show partitions'. I replace "/" with "#", which triggers character escaping but doesn't break partition path parsing.